packages/flutter_test/lib/src/binding.dart

```diff
@@ -1077,6 +1077,10 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
   @override
   int get microtaskCount => _currentFakeAsync!.microtaskCount;
 
+  /// In order to let animation get accurate time information during [pump],
+  /// set the flag to true. The default value is false for compatibility.
+  bool accuratePump = false;
+
```

Comment:
Is this because of the test failures in your earlier commits?

From what I can tell, one of those failures would be expected (the `widget_tester_test.dart "pumping pumpFrames"` test), and the other test is a customer test that I think is reasonable to make a change to or temporarily disable. I'll have to follow up with some people on whether the customer test change is acceptable or not.

If getting a change to the customer test is reasonable, I'd much prefer we get rid of this flag, as it doesn't actual do anything _except_ keep backwards compatibility for something we've pretty thoroughly determined is a bug.

Reply:
I'll followup again on Monday, but I'm hoping we can fix the failing customer test to prevent needing to add this flag.

Reply:
Yes, I add that flag because CI fails by widget_tester_test.dart and a consumer-test.

> and the other test is a customer test that I think is reasonable to make a change to or temporarily disable. I'll have to follow up with some people on whether the customer test change is acceptable or not.

Take your time :)

> If getting a change to the customer test is reasonable, I'd much prefer we get rid of this flag, as it doesn't actual do anything except keep backwards compatibility for something we've pretty thoroughly determined is a bug.

Totally agree! I also want to remove the flag

Reply:
Apologies for the delay, but I've reached out to the owner of the customer test that's failing and we think we can get a fix in the test that's compatible with this change without the flag. I'll work on getting that test fix in, while I'm doing that could you:

1) Remove the `accuratePump` flag
2) Fix the test in the `widget_tester_test.dart` file that was failing

After we get those failing tests cleared up, this should be good to go!

Reply:
Alright, the customer test should be fixed now. I tested it with the microsecond change locally to verify that the customer test works before _and_ after this change.

Reply:
Thanks :) Code done, waiting for CI